docs/_templates/class.rst

```diff
@@ -63,6 +63,15 @@
    a pre-selected object (for example, people) detected in each specified zone. The order of counts
    follows the order of ``zones``.
 
+.. |btm_midpoint| replace:: ``btm_midpoint`` (:obj:`list`): (x ,y) coordinates of the bottom middle point of detected
+   boxes. The order of btm_mipoint follows the order of `bboxes`.
```

Comment:
`btm_midpoint` is in L15, perhaps edit that?

Reply:
Edited L15 instead